lib/daemon_runner/retry_errors.rb

```diff
@@ -0,0 +1,18 @@
+require 'retryable'
+
+module DaemonRunner
+    #
+    # Retry Errors
+    #
+    class RetryErrors
+      class << self
+        def retry(retries: 3, exceptions: [Faraday::ClientError], &block)
+          Retryable.retryable({
+            on: exceptions,
```

Comment:
The default number of `tries` is [2](https://github.com/nfedyashev/retryable/blob/master/lib/retryable/configuration.rb#L34) so it'll only sleep the calculated number of seconds twice before throwing.